_topic_map.yml

```diff
@@ -105,6 +105,8 @@ Topics:
 - Name: The CI/CD methodology and practice
   File: cicd_gitops
   Distros: openshift-enterprise,openshift-webscale
+- Name: Using ArgoCD
```

Comment:
I was going to say that I didn't see how this fits under Architecture, but I do see that there is already a CI/CD assembly here, so I guess it makes sense to keep those together.

Reply:
> Just one comment, and for some reason the preview doesn't seem to be showing it under Architecture even though I see that's where it is in the topic map. So I'm not sure what to say about that.

It's rendering correctly in the Netlify and local preview for me, so I'm not sure what's gone wrong.

> I was going to say that I didn't see how this fits under Architecture, but I do see that there is already a CI/CD assembly here, so I guess it makes sense to keep those together.

That was Ali's reasoning, too. I'd be happier to remove the topic, but /shrug.

Reply:
The netlify preview now looks good to me. Not sure why it wasn't right when I looked at it an hour ago.